tests/unit/_experimental/test_plots.py

```diff
@@ -0,0 +1,1229 @@
+                        (LabeledBoundingBox((1, 1), (2, 2), "d"), ScoredLabeledBoundingBox((1, 1), (2, 2), "d", 0.9)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "d"), ScoredLabeledBoundingBox((1, 1), (2, 2), "d", 0.8)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "d"), ScoredLabeledBoundingBox((1, 1), (2, 2), "d", 0.7)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "d"), ScoredLabeledBoundingBox((1, 1), (2, 2), "d", 0.6)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "e"), ScoredLabeledBoundingBox((1, 1), (2, 2), "e", 0.99)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "e"), ScoredLabeledBoundingBox((1, 1), (2, 2), "e", 0.9)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "e"), ScoredLabeledBoundingBox((1, 1), (2, 2), "e", 0.01)),
+                    ],
+                    unmatched_gt=[
+                        (LabeledBoundingBox((1, 1), (2, 2), "a"), ScoredLabeledBoundingBox((1, 1), (2, 2), "b", 0.8)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "a"), ScoredLabeledBoundingBox((1, 1), (2, 2), "b", 0.1)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "b"), ScoredLabeledBoundingBox((1, 1), (2, 2), "c", 0.7)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "b"), ScoredLabeledBoundingBox((1, 1), (2, 2), "c", 0.2)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "c"), ScoredLabeledBoundingBox((1, 1), (2, 2), "d", 0.1)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "c"), ScoredLabeledBoundingBox((1, 1), (2, 2), "d", 0.1)),
+                        (LabeledBoundingBox((1, 1), (2, 2), "e"), None),
+                        (LabeledBoundingBox((1, 1), (2, 2), "e"), None),
+                        (LabeledBoundingBox((1, 1), (2, 2), "e"), None),
+                        (LabeledBoundingBox((1, 1), (2, 2), "e"), None),
+                    ],
+                    unmatched_inf=[
+                        ScoredLabeledBoundingBox((1, 1), (2, 2), "a", 0.8),
+                        ScoredLabeledBoundingBox((1, 1), (2, 2), "a", 0.8),
+                        ScoredLabeledBoundingBox((1, 1), (2, 2), "a", 0.8),
+                        ScoredLabeledBoundingBox((1, 1), (2, 2), "a", 0.8),
+                        ScoredLabeledBoundingBox((1, 1), (2, 2), "b", 0.1),
+                        ScoredLabeledBoundingBox((1, 1), (2, 2), "b", 0.1),
+                        ScoredLabeledBoundingBox((1, 1), (2, 2), "b", 0.1),
+                        ScoredLabeledBoundingBox((1, 1), (2, 2), "b", 0.1),
+                    ],
+                ),
+            ],
+            CurvePlot(
+                title="F1-Score vs. Confidence Threshold",
+                x_label="Confidence Threshold",
+                y_label="F1-Score",
+                curves=[
+                    Curve(
+                        x=[
```

Comment:
The current implementation produces `len(y_score)` (when there are < 401), but here `y_score` includes TP, FP, FN.  Does this make sense?  FNs don't have any valid scores (i.e. we are using `-1` here) so shouldn't there be only # of TP + FP thresholds?